fondat/aws/bedrock/resources/aliases.py

```diff
@@ -0,0 +1,71 @@
+"""Resource for managing agent aliases."""
+
+from collections.abc import Iterable
+from typing import Any, Mapping
+
+from fondat.aws.client import Config
+from fondat.pagination import Cursor
+from fondat.resource import resource
+from fondat.security import Policy
+
+from ..clients import agent_client
+from ..decorators import operation
+from ..pagination import decode_cursor, paginate
+
+
+@resource
+class AliasesResource:
+    """
+    Resource for managing agent aliases.
+    Provides access to list aliases and retrieve specific alias details.
+    """
+
+    __slots__ = ("_agent_id", "config_agent", "policies")
+
+    def __init__(
+        self,
+        agent_id: str,
+        *,
+        config_agent: Config | None = None,
+        policies: Iterable[Policy] | None = None,
+    ):
+        self._agent_id = agent_id
+        self.config_agent = config_agent
+        self.policies = policies
+
+    @operation(method="get", policies=lambda self: self.policies)
+    async def get(self, *, max_results: int | None = None, cursor: Cursor | None = None) -> Any:
```

Comment:
Should we return some other than `Any` ? Like an array of `AliasResource` or an array of `Alias` ?